### DISCUSSION TITLE
Connecting to Power BI Government Service (cloud)

### DISCUSSION THREAD

**Discussion by vansree** [sql-bi/Bravo#537] [General]
Hi,

How do I connect to Power BI Government Service (cloud)?

For reference, https://learn.microsoft.com/en-us/power-bi/enterprise/service-govus-overview

Commercial version: https://app.powerbi.com/
GCC: https://app.powerbigov.us/
GCC High: https://app.high.powerbigov.us/
DoD: https://app.mil.powerbigov.us/

Thanks

Sree

**Comment by albertospelta** [comment]
Hi @vansree, if your account is configured in more than one PBI cloud (e.g., Commercial and GCC) then when you sign in to Bravo you can choose which cloud to connect to, just as Power BI Desktop does. If your account is set up in only one PBI cloud, the environment is automatically chosen without prompting you to select it.

Bravo also supports the ability to [set the registry](https://github.com/microsoft/Federal-Business-Applications/tree/main/whitepapers/power-bi-registry-settings) to go directly to the right cloud endpoint for your environment bypassing the global discovery endpoint as explained here https://learn.microsoft.com/en-us/power-bi/enterprise/service-govus-overview#sign-in-to-power-bi-for-us-government and here https://www.youtube.com/watch?v=CTMG1OE17uk.


![image](https://user-images.githubusercontent.com/1957788/201864417-4d8f1c44-980b-4237-96cc-d30ba5151997.png)

![image](https://user-images.githubusercontent.com/1957788/201862721-00854ec8-97c7-4056-8742-65dcf028e2f7.png)

**Comment by vansree** [comment]
Thanks for the reply.

I am NOT able to and hence posted the issue.

Steps to reproduce:

1. Launch Bravo for Power BI - v1.0.0

2. Click "Connect to Power BI Service"

3. You will see "Open" window/dialog.
    Red under line on "Datasets on powerbi.com"
    (active tab)

    Message shows
	"You are not connected to Power BI - please sign in to proceed."

	"Sign In" button

4. Click "Sign In" button

5. You will see the dialog "Power BI"

	Enter your email address
![image](https://user-images.githubusercontent.com/49765261/201984021-3487edd7-9bbf-4724-b4ac-b20ba66c58bd.png)
	
	a small message will be shown briefly, "Checking..."

	Finally, you will get "Please enter a valid Power BI account."

...

Please let me know the next steps in resolving

**Reply by albertospelta** [reply]
@vansree, thank you for this feedback! I just opened a new issue here https://github.com/sql-bi/Bravo/issues/538 as this might be a bug.
Let's move the discussion there, thanks!

**Reply by vansree** [reply]
Sure and thanks for creating the issue and appreciated your reply.

**Reply by awatsky** [reply]
@vansree, as a fellow GCC user I was having this issue as well. @albertospelta has found a solution and posted it for you to test out. Take a look at the issue thread.
